### ISSUE TITLE
does it support decompiling linux so files?

### ISSUE THREAD

**Issue by He1my** [BoomerangDecompiler/boomerang#206]

**Comment by ceeac** [comment]
Yes and no - While Shared Object decompilation itself is supported for 32-bit SO files, it is untested as of now. Also, only ELF files are supported, not old formats like COFF or a.out.
